core-kit/next/sys-devel/gnuconfig/templates/gnuconfig.tmpl

```diff
@@ -0,0 +1,37 @@
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="5"
+
+inherit eutils
+
+DESCRIPTION="{{desc}}"
+HOMEPAGE="{{homepage}}"
+SRC_URI="{{src_uri}}"
+
+KEYWORDS="*"
+S="${WORKDIR}"
+
+LICENSE="GPL-2"
+SLOT="0"
+IUSE=""
+
+src_unpack() {
+	unpack ${A}
+}
+
+src_prepare() {
+	use elibc_uclibc && sed -i 's:linux-gnu:linux-uclibc:' testsuite/config-guess.data #180637
+}
+
+src_compile() { :;}
+
+src_test() {
```

Comment:
This could be removed considering that the USE flag `test` is not present in IUSE.